proto/aserto/directory/schema/v2/group.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package aserto.directory.schema.v2;
+
+option go_package = "github.com/aserto-dev/go-directory/aserto/directory/schema/v2;schema";
+
+// Properties of "group" objects.
+message GroupProperties {
+    string connection_id = 1;  // ID of the IDP connection the group is associated with.
```

Comment:
As connection_id is not mandatory, do we need to be able to check for nullability, in other words mark the property `optional` ?

Reply:
I think the check for nullability is important in cases where the default value could have a valid meaning. If the empty string was a valid connection id then I think it would make sense to make it nullable (i.e. `optional`) but I don't think it's necessary in this case.